Model/Config.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Elgentos\Kiyoh\Model;
+
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Store\Model\StoreManagerInterface;
+
+class Config {
+
+    /**
+     * @var StoreManagerInterface
+     */
+    private StoreManagerInterface $storeManager;
+
+    const PUBLICATION_URL = 'https://www.kiyoh.com/v1/publication/review/external/location/statistics?locationId=';
+
+    public function __construct(
+        StoreManagerInterface $storeManager
+    ) {
+        $this->storeManager = $storeManager;
+    }
+
+    /**
+     * @throws NoSuchEntityException
+     */
+    public function getApiKey(): mixed {
+        return $this->storeManager->getStore()->getConfig(
```

Comment:
Since this can be set per store, please make sure you can fetch the value per store view. The same goes for the `getLocationId()`. You can rewrite this to this:

```php
use Magento\Store\Model\ScopeInterface; // Should be at the top of your file

return $this->storeManager->getStore()->getConfig(
    'kiyoh_settings/general/api_key',
    ScopeInterface::SCOPE_STORE
)
```